include/constants/pokemon.h

```diff
@@ -80,7 +81,7 @@
 #define STAT_ACC     6 // Only in battles.
 #define STAT_EVASION 7 // Only in battles.
 
-#define NUM_EV_STATS     NUM_STATS - 1 // excludes HP
+#define NUM_NATURE_STATS NUM_STATS - 1 // excludes HP
```

Comment:
hmmm? why nature instead of ev?

Reply:
HP has an EV, but not a Nature modification. It's named incorrectly.